subsys/net/lib/nrf_cloud/src/nrf_cloud_rest.c

```diff
@@ -1154,3 +1158,144 @@ int nrf_cloud_rest_jitp(const sec_tag_t nrf_cloud_sec_tag)
 
 	return ret;
 }
+
+int nrf_cloud_rest_send_location(struct nrf_cloud_rest_context *const rest_ctx,
+	const char *const device_id, const char *const nmea_sentence)
+{
+	__ASSERT_NO_MSG(rest_ctx != NULL);
+	__ASSERT_NO_MSG(device_id != NULL);
+	__ASSERT_NO_MSG(nmea_sentence != NULL);
+
+	int err = -ENOMEM;
+	char *json_msg = NULL;
+
+	(void)nrf_cloud_codec_init();
+
+	cJSON *root_obj = json_create_req_obj(NRF_CLOUD_JSON_APPID_VAL_GPS,
+					     NRF_CLOUD_JSON_MSG_TYPE_VAL_DATA);
+
+	if (cJSON_AddStringToObjectCS(root_obj, NRF_CLOUD_JSON_DATA_KEY, nmea_sentence) == NULL) {
```

Comment:
You compare cJSON_AddStringToObjectCS result against NULL here, but later compare against cJSON_True -- is that intentional?

Reply:
not intentional, fixed.